src/EStimLibrary/Extensions/HardwareInterfaces/ContactGroup.cs

```diff
@@ -13,7 +13,8 @@ public class ContactGroup : NeuralInterfaceHardware
 
     public ContactGroup(int numContacts) : base()
```

Comment:
Add doc comments to each class and method as you're going, so for this constructor might be:
```
/// <summary>
/// Create a contact group with a positive integer number of contacts.
/// </summary>
/// <param name="numContacts">TODO</param>
/// <exception cref="ArgumentException">TODO</exception>
```